src/server/middleware.ts

```diff
@@ -121,15 +125,21 @@ export function createCustomManifestMiddleware(): RequestHandler {
             defaultTop: Number.parseInt(req.query.defaultTop, 10) || 605,
             defaultWidth: Number.parseInt(req.query.defaultWidth, 10) || 860,
             defaultHeight: Number.parseInt(req.query.defaultHeight, 10) || 605,
-            licenseKey: defaultConfig.licenseKey
+            licenseKey: defaultConfig.licenseKey,
+            shortcut: req.query.shortcutName && {
```

Comment:
Can we put `|| undefined` on the end of this? Just thinking of the case if we don't provide a shortcut

Reply:
I think this will automatically be evaluated to || undefined in case shortcutName is falsy. but sure for clarity